File: Ruby_/entradaSaida/Exemplo2.rb
# frozen_string_literal: true

print "Digite a quantidade de bolos por ovo: "
ovosBolo = gets.to_i
print "Digite a quantidade de ovos que você tem: "
qtdOvos = gets.to_i

qtdBolos = qtdOvos / ovosBolo

puts("Com #{qtdOvos} ovos é possível fazer #{qtdBolos} bolos")

File: Kotlin/entradaSaida/Exemplo2.kt
package entradaSaida

fun main(){
    print("Digite a quantidade de bolos por ovo: ")
    val ovosBolo = readln().toInt()
    print("Digite a quantidade de ovos que você tem: ")
    val qtdOvos = readln().toInt()

    val qtdBolos = qtdOvos / ovosBolo

    println("Com $qtdOvos ovos é possível fazer $qtdBolos bolos")
}